arches/app/etl_modules/save.py

```diff
@@ -123,3 +127,20 @@ def save_to_tiles(userid, loadid):
             )
             return {"success": False, "data": "saved"}
 
+
+def get_resourceids_from_search_url(search_url, user=None):
+    request = HttpRequest()
+    request.user = user
+    request.method = "GET"
+    request.GET["export"] = True
+    validate = URLValidator()
+    try:
+        validate(search_url)
+    except:
+        raise
+    params = parse_qs(urlsplit(search_url).query)
+    for k, v in params.items():
+        request.GET.__setitem__(k, v[0])
+    response = search_results(request)
```

Comment:
Can this be replaced with a usage of `reverse` so that a project's override of search takes precedence?
Here's an [example](https://github.com/archesproject/arches/blob/dev/7.6.x/arches/app/search/search_export.py#L165)